src/Tools/_framework/Paths/PortfolioActivityEditor.jsx

```diff
@@ -230,6 +233,12 @@ export async function loader({ params }) {
       platform = "Mac";
     }
 
+    const allDoenetmlVersionsResp = await axios.get(
```

Comment:
It is silly how this api is called every time a portfolio activity editor is opened even though we need it only for the control panel.